src/BasePresenter.php

```diff
@@ -166,4 +167,20 @@ public function __toString()
     {
         return (string) $this->wrappedObject;
     }
+
+    /**
+     * Ensures the presenter will keep its behavior when json_encode is called on it.
+     *
+     * @return false|mixed|string
+     */
+    public function jsonSerialize()
+    {
+        $attributes = [];
+
+        foreach ($this->wrappedObject->getAttributes() as $attribute => $value) {
+            $attributes[$attribute] = $this->$attribute;
```

Comment:
What about recursively encoding the attributes?

Reply:
Do laravel's model implement jsonSeralize?

Reply:
> 
> 
> Do laravel's model implement jsonSeralize?

Yes. https://laravel.com/api/7.x/Illuminate/Database/Eloquent/Model.html#method_jsonSerialize just came here because I could use this feature on a project.  I've just called `$this->getWrappedObject()->jsonSerialize();` where necessary